spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -33,6 +34,8 @@
         and_return(socure_secret_key_queue)
       allow(IdentityConfig.store).to receive(:socure_enabled).
         and_return(socure_enabled)
+      allow(NewRelic::Agent).to receive(:notice_error)
```

Comment:
should this only be stubbed for the tests that is actually hitting it? 🤔 otherwise, i think we could have it called mistakenly but notice it